.gitattributes

```diff
@@ -0,0 +1 @@
+*.sh -crlf
```

Comment:
nit: can you add a new line to the end of this file?

Reply:
Getting the error message for refernece

Reply:
added